src/Http/Http/src/HttpContextAccessor.cs

```diff
@@ -5,10 +5,16 @@
 
 namespace Microsoft.AspNetCore.Http
 {
+    /// <summary>
+    /// Represents the properties and methods used to access the current HTTP context. 
```

Comment:
```suggestion
    /// Provides an implementation of <see cref="IHttpContextAccessor" /> based on the current execution context.
```

Since this is a doc for the class, we can be a little more concrete here.